### PR TITLE
OTG Test Update - RT-5.4 Fix the ATE destination device name

### LINKED ISSUE #1273: OTG: RT-5.4: Aggregate Forwarding Viable

### PR DESCRIPTION
**Pull request by octpetre**
This fix will ensure that otg configurations will get pushed
Resolves #1273

### CODE REVIEW

File: feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go [hunk 1]
```diff
@@ -333,7 +333,7 @@ func (tc *testArgs) configureATE(t *testing.T) {
 	tc.top.Ports().Add().SetName(p0.ID())
 	d0 := tc.top.Devices().Add().SetName(ateSrc.Name)
 	srcEth := d0.Ethernets().Add().SetName(ateSrc.Name + ".Eth").SetMac(ateSrc.MAC)
-	srcEth.Connection().SetPortName(p0.ID())
+	srcEth.Connection().SetChoice(gosnappi.EthernetConnectionChoice.PORT_NAME).SetPortName(p0.ID())
```
Comment (jasdeep-hundal) [inline feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go:336]: Gosnappi automatically sets the choice in the `SetPortName` function, adding this should not be necessary.

File: feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go [hunk 2]
```diff
@@ -380,9 +380,9 @@ func (tc *testArgs) configureATE(t *testing.T) {
 		autoNegotiate.SetRsFec(false)
 	}
 
-	dstDev := tc.top.Devices().Add().SetName(agg.Name())
+	dstDev := tc.top.Devices().Add().SetName(agg.Name() + ".dev")
 	dstEth := dstDev.Ethernets().Add().SetName(ateDst.Name + ".Eth").SetMac(ateDst.MAC)
-	dstEth.Connection().SetLagName(agg.Name())
+	dstEth.Connection().SetChoice(gosnappi.EthernetConnectionChoice.LAG_NAME).SetLagName(agg.Name())
```
Comment (jasdeep-hundal) [inline feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go:385]: Same comment as above, Gosnappi automatically sets the choice when calling `SetLagName`

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go (hunk 2 above)
Comment (octpetre) [reply feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go:385]: Removed.

File: feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go (hunk 1 above)
Comment (octpetre) [reply feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go:336]: Removed

**Review by jasdeep-hundal — APPROVED** [review]

**Pull request merged**